rasa/shared/utils/common.py

```diff
@@ -65,6 +65,11 @@ def module_path_from_instance(inst: Any) -> Text:
     return inst.__module__ + "." + inst.__class__.__name__
 
 
+def module_path_from_class(clazz: Type) -> Text:
```

Comment:
do we actually need it in `rasa.shared`?

Reply:
probably not - but it's so close in functionality to `module_path_from_instance` that it seemed weird to have it anywhere else. I don't think we should actually have any utility functionality in shared.

Reply:
then let's move it out 👍🏻